src/button.tsx

```diff
@@ -517,6 +519,7 @@ export const ButtonLink = React.forwardRef<
                 {props.children}
             </Text2>
         ),
+        disabled: props.disabled,
```

Comment:
You should also read the form state and if sending, automatically disable the button.

See the button implementation

Reply:
I see we are looking for form status in TextLinks too, so it makes sense to be consistent here

Reply:
done